spec/requests/sites_spec.rb

```diff
@@ -10,21 +10,21 @@
   end
 
   describe "GET index" do
-    it "redirects to /2019/locations" do
+    it "redirects to /2020/locations" do
```

Comment:
may be just stick 2020 in a variable to be shared for the others

current_recruitment_cycle_year = 2020

Reply:
Not sure variables can be used in `it` names :thinking: I'd go more generic with "Redirects to /locations for the current recruitment cycle", then it doesn't need to be updated for every year